projects/ui/dialog/body/dialog-header.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { cva } from 'class-variance-authority'
+
+export const dialogHeaderStyles = cva(
+  [
+    'flex',
+    'flex-col',
+    'space-y-1',
+    'px-6',
+    'pb-3',
+    'pt-6',
+    'bg-surface/95',
+    'backdrop-blur',
+    'supports-[backdrop-filter]:bg-surface/60'
```

Comment:
Maybe it's better to have a usual background?

Reply:
Dunno, looks poor imo.. Compare
![image](https://github.com/user-attachments/assets/6b09e621-0fce-43e4-8e6f-365521829189)
![image](https://github.com/user-attachments/assets/a8fc72a6-1d17-4d2f-968c-0ab8ed00554a)

Reply:
I _generally_ follow Apple's design guidelines when deciding on how things should look like here. And they do use blur in similar cases. Looks especially nice in dark mode
![image](https://github.com/user-attachments/assets/1e82bb9c-a277-491e-a7ac-5a77d846d642)